.github/workflows/merge-to-stage.js

```diff
@@ -143,14 +142,6 @@ const merge = async ({ prs, type }) => {
       console.log(`Current number of PRs merged: ${existingPRCount}`);
       const prefix = type === LABELS.zeroImpact ? ' [ZERO IMPACT]' : '';
       body = `-${prefix} ${html_url}\n${body}`;
-      await slackNotification(
```

Comment:
This was removed from merge-to-stage, what about removing the same from merge-to-main?